django/forms/models.py

```diff
@@ -756,7 +756,7 @@ def save_new(self, form, commit=True):
         """Save and return a new model instance for the given form."""
         return form.save(commit=commit)
 
-    def save_existing(self, form, instance, commit=True):
+    def save_existing(self, form, obj, commit=True):
```

Comment:
This is an internal API but I'd add a small release note in the [_"Backwards incompatible"_](https://docs.djangoproject.com/en/dev/releases/5.0/#miscellaneous) section, e.g.
```
* The ``instance`` argument of the undocumented
  ``BaseModelFormSet.save_existing()`` method is renamed to ``obj``.
```

Reply:
done